### DISCUSSION TITLE
Why does RPCResult or .rpc not expose msg_id?

### DISCUSSION THREAD

**Discussion by iostreamer-X** [sitano/maelstrom-rust-node#6] [Q&A]
I was hoping to get a handle of what msg_id was sent through rpc but there doesn't seem to be a straightforward way to get that. 
I was hoping to understand if this is intended behaviour and if I am missing something if I am wanting that?

Thank you!

**Comment by sitano** [comment]
hi @iostreamer-X . I don't think I meant anything. Probably its just I didn't think you can do with it anything as API does not expose any other functions that wants Msg_id. Anyway, I think we can easily make it public / or provide a function `.msg_id()`.
